js/src/launchAsync.ts

```diff
@@ -256,19 +291,274 @@ export function launchAsync(token: string, subdomain: string, content: Content,
     });
 }
 
-export function close(): void {
-    window.postMessage('ImmersiveReader-Exit', '*');
-}
+/**
+ * Launch the Immersive Reader within an iframe.
+ * @param options Options for configuring the look and feel of the Immersive Reader.
+ * @return A promise that resolves with a LaunchWithoutContentResponse to provide the S2S response
+ */
+export function launchWithoutContentAsync(options?: Options): Promise<LaunchWithoutContentResponse> {
```

Comment:
>launchWithoutContentAsync [](start = 16, length = 25)

This is copied from the 004 sdk - only change is postLaunchOperations on line 417